tests/e2e/support/commands.js

```diff
@@ -118,7 +118,7 @@ Cypress.Commands.add('setupGameAsSpectator', () => {
   cy.visit('/');
   cy.signupPlayer(username, validPassword);
   cy.vueRoute('/');
-  cy.createGamePlayer({ gameName: 'Test Game', isRanked: false }).then((gameData) => {
+  cy.createGamePlayer({ gameName: 'Spectator Game', isRanked: false }).then((gameData) => {
```

Comment:
Updated this to make it easier to tell the difference between the spectator game and the one we're playing